test/libsolidity/GasCosts.cpp

```diff
@@ -70,13 +70,17 @@ BOOST_AUTO_TEST_CASE(string_storage)
 
 	if (Options::get().evmVersion() <= EVMVersion::byzantium())
 		CHECK_GAS(134435, 130591, 100);
+	else if (Options::get().useABIEncoderV2)
```

Comment:
Why isn't this the first if?

Reply:
Because it is only correct on constantinople.

Reply:
The test assumes that you do not run the v2 tests on pre-constantinople.

Reply:
Ok, can you add a comment in both places this the reason so that others' wont move it around?